### PR TITLE
10-final_snischuk

### PR DESCRIPTION
**Pull request by snischuk**

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @snischuk, please add a link to your deployed project
> Зробіть пуллреквест із посиланням на ваш задеплоїний проєкт

File: homeworks/max.snischuk_snischuk/PokemonProject/styles/base.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+:root {
+  --color-dark: #221F1F;
+  --color-light: #FFF;
+  --color-accent: #EF4934;
+
+  --padding-container: 16px;
+}
+
+@media (min-width: 769px) {
+
```
Comment (vladyslav-yermolin-moc) [inline homeworks/max.snischuk_snischuk/PokemonProject/styles/base.css:31]: please add a `min-width`
> Зробіть резінову верстку:
> мінімальна ширина контенту - 400px (меньше - зʼявляється скрол)

File: homeworks/max.snischuk_snischuk/PokemonProject/index.html [hunk 2]
```diff
@@ -32,7 +32,7 @@
         </header>
         <main class="about">
             <div class="about__container">
-                <img class="about__image" src="./images/about.png" alt="pokemon">
+                <img class="about__image" src="./images/about.jpeg" alt="pokemon">
```
Comment (vladyslav-yermolin-moc) [inline homeworks/max.snischuk_snischuk/PokemonProject/index.html:35]: As a recommendation: the language of your site is English, but you have the Ukrainian error messages. Looks inconsistent.
image

Comment (vladyslav-yermolin-moc) [reply homeworks/max.snischuk_snischuk/PokemonProject/index.html:35]: The height of the input is different for normal and focus states - it leads to `content-jumping`
image
Screenshot at Dec 14 19-19-45

File: homeworks/max.snischuk_snischuk/PokemonProject/pokemons.html [hunk 3]
```diff
@@ -44,7 +44,8 @@
                     <fieldset class="filters__group">
                         <input class="filters__input" id="all" type="radio" name="radio-options" value="all">
                         <label class="filters__custom filters__custom--radio" for="all">All</label>
-                        <input class="filters__input" id="favorites" type="radio" name="radio-options" value="favorites">
+                        <input class="filters__input" id="favorites" type="radio" name="radio-options"
```
Comment (vladyslav-yermolin-moc) [inline homeworks/max.snischuk_snischuk/PokemonProject/pokemons.html:47]: Your logic of filtering broken a bit:
1. Check Small checkbox
2. Type s in the search
3. I see big pokemons which go to hidden when the input lost focus (any click)

image

Comment (vladyslav-yermolin-moc) [reply homeworks/max.snischuk_snischuk/PokemonProject/pokemons.html:47]: It is good, that you used `featured` prop of the Pokemons for the filtering by radio, but not a user can't understand, what is going on, when they hit `Favorite` radio.
Would be good to add some visual representation to the card (e.g. heart icon or another badge)
image

File: homeworks/max.snischuk_snischuk/PokemonProject/calculator.html [hunk 4]
```diff
@@ -47,6 +47,51 @@ <h1 class="calculator__title">Calculate pokemons</h1>
                 <p class="calculator__time" data-math-time></p>
             </div>
         </main>
+        <footer class="footer">
```
Comment (vladyslav-yermolin-moc) [inline homeworks/max.snischuk_snischuk/PokemonProject/calculator.html:50]: Same content jumps I can see on this page, when both inputs are filled and I hit `=` button

**Review by vladyslav-yermolin-moc — CHANGES_REQUESTED** [review]

**Comment by snischuk** [comment]
@vladyslav-yermolin-moc Very useful comments!) Thanks for review!)
P.S sorry for pokemons-filters, i forgot that main branch is without "9-async_snischuk" branch)

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @snischuk, please deploy you project and add a link to it here. Also please move your project files to `docs` folder and add a link to `docs/index.html` as described in https://github.com/MastersAcademy/fe-course-basic-2023/issues/423

**Comment by vladyslav-yermolin-moc** [comment]
Hey, @snischuk ,I've merged your task 9 pr, so, you have some time to finish your project and deploy it to our[ "Hall of Fame" ](https://mastersacademy.github.io/fe-course-basic-2023/)

**Pull request closed without merging**